src/Breadcrumbs/Recorder.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+namespace Bugsnag\Breadcrumbs;
+
+use Iterator;
+
+class Recorder implements Iterator
```

Comment:
Nope, but it would mean that the recorder is natively countable, so we could call `count($recorder)`, and it would return an integer between 0 and 25 representing the number of breadcrumbs we have stored.